integrations/gitlab/gitlab_integration/events/setup.py

```diff
@@ -124,30 +144,45 @@ def setup_system_listeners(gitlab_clients: list[GitlabService]) -> None:
         system_event_handler.add_client(gitlab_service)
 
 
+def appened_events_to_groups(
+    groups_hooks_events_override: dict[str: dict[str: list[str]]] | None,
```

Comment:
can we turn the `dict[str: dict[str: list[str]]]` to readable object? use pydantic to define the object

Reply:
specifying a class for that will allow us to maintain more readable code and easier to add more things other than events in a readable way.

Reply:
done